src/platform/PICO/target/RP2350B/target.h

```diff
@@ -247,7 +247,7 @@
 #define SPI1_SDO_PIN         PA27
 
 #define SDCARD_CS_PIN        PA25
```

Comment:
Do the same for SDCARD_CS_PIN ?

Reply:
Once the HELLBENDER is merged in config - absolutely - I will do a few more changes here. The problem at the minute is that the Flash CS is not 25 it is 0. So this is to fix up for the Hellbender to build correctly with it defined.